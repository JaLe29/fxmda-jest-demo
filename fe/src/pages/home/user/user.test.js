import React from 'react';
import {shallow} from 'enzyme';

import User from './user';

const mockProps = {
	name: 'Jarda',
	age: 20,
}

describe('User', () => {
	it('renders user component', () => {
		const wrapper = shallow(<User {...mockProps} />);
		expect(wrapper).toMatchSnapshot();
	});

	it('should render with adult className', () => {
		const wrapper = shallow(<User {...mockProps} age={10} />);
		expect(wrapper.find('div').at(0).hasClass('user-10')).toEqual(true);
	});
});