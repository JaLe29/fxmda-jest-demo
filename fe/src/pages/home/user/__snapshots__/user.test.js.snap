// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User renders user component 1`] = `
<div
  className="user-20"
>
  <div>
    Name: 
    Jarda
  </div>
  <div>
    Age: 
    20
  </div>
</div>
`;
